pkg/api/core/cloudprovider/aws/identitymanager.go

```diff
@@ -0,0 +1,128 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/gosimple/slug"
+
+	"github.com/oslokommune/okctl/pkg/api"
+	"github.com/oslokommune/okctl/pkg/api/okctl.io/v1alpha1"
+	"github.com/oslokommune/okctl/pkg/cfn"
+	"github.com/oslokommune/okctl/pkg/cfn/components"
+	"github.com/oslokommune/okctl/pkg/cognito"
+)
+
+const (
+	// DefaultCloudFrontHostedZoneID is the default hosted zone id for all cloud front distributions
+	DefaultCloudFrontHostedZoneID = "Z2FDTNDATAQYW2"
```

Comment:
Will this be a problem for multiple hosted zones on same AWS account?

Reply:
See: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-route53-aliastarget.html

> CloudFront distribution
Specify Z2FDTNDATAQYW2. This is always the hosted zone ID when you create an alias record that routes traffic to a CloudFront distribution.